ngraph/test/backend/deformable_convolution_opset8.in.cpp

```diff
@@ -0,0 +1,2940 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Is there really no chance to reuse existing tests from deformable_convolution.in.cpp?

Reply:
we can parameterize most of the tests, but it looks like it will require serious rework
parameterization approach only for version (v1, v8) will not bring much improvement, so we need to parameterize inputs while maintaining the same test coverage.
it's better to do it as a separate task, if it's required